src/services/shopify-client.ts

```diff
@@ -0,0 +1,25 @@
+export enum SubscriberListState {
+  ready,
+  synchronizing_contacts,
+}
+
+export interface SubscriberList {
+  name: string;
+  id: number;
+  amount_subscribers: number;
+  state: SubscriberListState;
+}
+
+export interface ShopifyConnection {
+  shop_name: string;
+  synchronization_date: Date | null;
+  list: SubscriberList;
+  connected: boolean;
+}
+
+export interface ShopifyClient {
+  getShopifyData(email: string, apiKey: string): Promise<ShopifyConnection>;
```

Comment:
If I am not wrong, there could be more than a shop connected, what do you think about:

```suggestion
  getShopifyData(email: string, apiKey: string): Promise<ShopifyConnection[]>;
```